src/app.js

```diff
@@ -1,27 +1,33 @@
 const log = false;
 
-const showCustomPage = ({ customNewTabUrl }) => {
+const showCustomPage = ({ customNewTabUrl, customNewTabTitle }) => {
 	log && console.debug( '[showCustomPage] init', { customNewTabUrl } );
 
+	if( customNewTabTitle ){
```

Comment:
Please add a space between the `if` keyword and the bracket to match the coding style. I've pushed some updated linting rules to cover this (use `yarn lint` to check).

Reply:
I guess I should also add a space between `){`, to conform with the next `if` statement.